src/components/TouchableDismissKeyboard/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** The children which should be contained in this wrapper component. */
+    children: PropTypes.node.isRequired,
+};
+
+const TouchableDismissKeyboard = props => (
+    <>
+        {props.children}
+    </>
+);
```

Comment:
Can this be simplified further (not sure if this works totally):

```suggestion
const TouchableDismissKeyboard = props => props.children;
```

Reply:
Ooo that looks nice, will try it out!